pom.xml

```diff
@@ -370,6 +370,22 @@
         </plugins>
     </build>
 
+    <reporting>
+        <plugins>
+            <plugin>
+                <artifactId>maven-project-info-reports-plugin</artifactId>
+                <version>2.9</version>
+                <reportSets>
+                    <reportSet>
+                        <configuration>
+                            <skip>true</skip>
```

Comment:
Is it worth a comment explaining why we're skipping them?

Reply:
Done